### ISSUE TITLE
Adding Pages through plugin

### ISSUE THREAD

**Issue by SzymonPawliszyn** [FBarrca/obsidian-handwritten-notes#10]
Hi!
While creating handwritten note pdf document through yours plugin inside Obsidian, would it be possible to add option of how many pages to create?

**Comment by FBarrca** [comment]
Yes, this is something I've considered. It should be possible using PDF-lib. I'm currently quite busy and won't work on this feature rn, as you can easily make new pages on Xodo, but you are welcome to work on it yourself and PR.

**Comment by RyotaUshio** [comment]
Hi! My beta plugin [PDF++](https://github.com/RyotaUshio/obsidian-pdf-plus) supports this. Check it out if you're interested.
